models/ticket_history/int_zendesk__field_history_enriched.sql

```diff
@@ -0,0 +1,25 @@
+with ticket_field_history as (
+
+    select *
+    from {{ ref('stg_zendesk__ticket_field_history') }}
+
+), users as (
+    select *
+    from {{ ref('stg_zendesk__user') }}
+
+), final as (
+    select
+        ticket_field_history.*,
+        case when ticket_field_history.user_id = -1
+            then 'zendesk_auto_change'
```

Comment:
The `-1` fields are the Zendesk automated changes. Let me know if you would prefer a different name for these.